x/streamer/keeper/keeper.go

```diff
@@ -81,6 +105,18 @@ func (k Keeper) CreateStream(ctx sdk.Context, coins sdk.Coins, records []types.D
 		return 0, fmt.Errorf("epoch identifier does not exist: %s", epochIdentifier)
 	}
 
+	pointerExists, err := k.HasEpochPointer(ctx, epochIdentifier)
+	if err != nil {
+		return 0, fmt.Errorf("check epoch pointer existence: %w", err)
+	}
+
+	if !pointerExists {
```

Comment:
can set on genesis with the avaialble `epochs`, as the epoch list is immutable
than u can assume `EpochPointer` exists for all epochs